ui.frontend/src/main/webpack/components/_helloworld.scss

```diff
@@ -1,7 +1,9 @@
 // Best practice:
 // For a good separation of concerns, don't rely on the DOM element names,
 // but on dedicated CSS selectors to identify all elements
-.cmp-helloworld {}
+.cmp-helloworld {
```

Comment:
this is out of the scope of the story.
Please clarify why this is necessary

Reply:
Changes removed